frontend-react/src/components/Table/TableFilters.tsx

```diff
@@ -102,100 +162,171 @@ function TableFilters({
     const resetHandler = useCallback(
         (e: FormEvent) => {
             e.preventDefault();
-            if (formRef.current) {
```

Comment:
Manipulating the DOM like this is cumbersome, which is why I'm just re-rendering the `<form/>` with `setReset(reset + 1);`